vampire_survivors/src/highscore.py

```
@@ -0,0 +1,23 @@
+    def load_highscore(self):
+        try:
+            with open('highscore.json', 'r') as f:
+                self.highscore = json.load(f)
+        except FileNotFoundError:
+            self.highscore = 0
+
+    def save_highscore(self):
+        with open('highscore.json', 'w') as f:
+            json.dump(self.highscore, f)
+
+    def set_highscore(self, score):
```

Ruff (0.12.2)

Missing return type annotation for public function `set_highscore`

Add return type annotation: `None`

(ANN201)